api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -31,7 +31,7 @@
  * <p>A ConfigSource will get picked up via the
  * {@link java.util.ServiceLoader} mechanism and must get registered via
  * META-INF/services/javax.config.spi.ConfigSource</p>
- * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}.
+ * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}. << How?
```

Comment:
It is not clear to me how a custom source can be added programmatically to existing sources?
AFAIK, only defining a completely new config is possible, but that wouldn't change the config returned by `ConfigPrvider.getConfig()`. Is there any way to change this config programmatically? Or only via service loader?